enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.tsx

```diff
@@ -20,12 +21,12 @@ export const EditorButton = ({
       <ActionIcon
         data-testid="notebook-button"
         size="lg"
-        c={isOpen ? "text-white" : "text-dark"}
-        color="brand"
```

Comment:
I guess this would be overridden by `c` anyway? Is that why you completely remove it?

Reply:
@WiNloSt by the className prop, yes.